File: src/main/java/com/Khata/Khata/Controller/LoginController.java
package com.Khata.Khata.Controller;

import com.Khata.Khata.Entity.LoginDTO;
import com.Khata.Khata.Entity.User;
import com.Khata.Khata.Service.LoginService;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;

@Controller
public class LoginController {

    private LoginService loginService;

    LoginController(LoginService loginService)
    {
        this.loginService = loginService;
    }

    @PostMapping("/login")
    public User Login(@RequestBody LoginDTO credentials)
    {

    }
}

File: classtracker/src/main/kotlin/com/classtrackeer/classtracker/Controller/loginController.kt
package com.classtrackeer.classtracker.Controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class loginController {

    @GetMapping("/login")
    fun showLoginPage(): String {
        return "login"
    }
    @GetMapping("/signup")
    fun showSignupPage() : String {
        return "signup"
    }
    @GetMapping("/dashboard")
    fun showDashboard() : String {
        return "dashboard"
    }
}